modules/oe_content_entity/modules/oe_content_entity_organisation/modules/oe_content_organisation_reference/config/install/field.field.oe_contact.oe_organisation_reference.oe_node_reference.yml

```diff
@@ -0,0 +1,33 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_contact.oe_node_reference
+    - node.type.oe_organisation
+    - oe_content_entity_contact.oe_contact_type.oe_organisation_reference
+  module:
+    - composite_reference
+    - entity_reference_revisions
+third_party_settings:
+  composite_reference:
+    composite: false
```

Comment:
Should we disable composite_reference?

Reply:
In this case, yes. We don't want organisations to be deleted.